database/src/main/resources/db/migration/V1_25__Add_gas_fee_volumes.sql

```diff
@@ -0,0 +1,98 @@
+-- Host future and previous migrated data
+CREATE TABLE IF NOT EXISTS tx_gas_cache
+(
+    id           BIGSERIAL PRIMARY KEY,
+    hash         VARCHAR(64) UNIQUE    NOT NULL,
+    tx_timestamp TIMESTAMP             NOT NULL,
+    gas_wanted   INT                   NOT NULL,
+    gas_used     INT                   NOT NULL,
+    fee_amount   DOUBLE PRECISION      NOT NULL,
+    processed    BOOLEAN DEFAULT false NOT NULL
+);
+
+CREATE INDEX IF NOT EXISTS tx_gas_cache_tx_timestamp_idx ON tx_gas_cache (tx_timestamp);
+CREATE INDEX IF NOT EXISTS tx_gas_cache_tx_gas_used_idx ON tx_gas_cache (gas_used);
+CREATE INDEX IF NOT EXISTS tx_gas_cache_tx_processed_idx ON tx_gas_cache (processed);
+
+-- Aggregate daily gas volume
+CREATE TABLE IF NOT EXISTS tx_gas_fee_volume_day
```

Comment:
I dont think this table is necessary, if we have the hourly table.

Reply:
No we do not need it. I created the table so we don't have to aggregate daily on the fly.

Reply:
okay.